django-backend/fecfiler/transactions/schedule_f/models.py

```diff
@@ -0,0 +1,54 @@
+from uuid import uuid4
+from django.db.models import (
+    TextField,
+    ForeignKey,
+    BooleanField,
+    DecimalField,
+    CASCADE,
+    DateField,
+    UUIDField,
+    Model,
+)
+
+
+class ScheduleF(Model):
+    id = UUIDField(
+        default=uuid4,
+        editable=False,
+        primary_key=True,
+        serialize=False,
+        unique=True,
+    )
+    filer_designated_to_make_coordianted_expenditures = BooleanField(
+        null=True, blank=True
+    )
+    designating_committee_account = ForeignKey(
```

Comment:
This needs to be a contact.  `CommitteeAccount` is a committee's account on our website rather than an actual FEC committee.  (An FEC committee would make a committee account on FECfile+)